### ISSUE TITLE
Parking_sensors - deployment to Azure failed.

### ISSUE THREAD

**Issue by dmitribodiu** [Azure-Samples/modern-data-warehouse-dataops#360]
After executing deploy.sh, I received this error:
ERROR: Error occurred in request., InvalidSchema: No connection adapters were found for 'C:/Program Files/Git/subscriptions/fdb69e38-5ba1-4a9f-a867-1de43543997e/resourceGroups/mdwdo-park-fkfkf-dev-rg/providers/Microsoft.Storage/storageAccounts/mdwdostordevfkfkf/providers/Microsoft.Authorization/roleDefinitions?$filter=roleName%20eq%20%27Storage%20Blob%20Data%20Contributor%27&api-version=2018-01-01-preview'

![MINGW64dProjectsdatawarehouse-importe2e_samplesparking_sensors](https://user-images.githubusercontent.com/6432848/123122006-51cf2680-d44e-11eb-90f6-8a7f600bf4fa.jpg)

I didn't use dev containers

**Comment by devlace** [comment]
HI @dmitribodiu. This error looks like its related to the fact that you are running az cli using git bash: https://github.com/Azure/azure-cli/issues/16317

TLDR: the fix seems to be setting MSYS_NO_PATHCONV=1 and ensuring you have the latest az cli version.

**Comment by dmitribodiu** [comment]
Indeed, thanks a lot, it helped!
